src/conponents/Social.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faTwitter, faSquareFacebook, faSquareInstagram, faSquareGithub,
+} from '@fortawesome/free-brands-svg-icons';
+
+const SocialCard = () => (
+  <div className="social-card">
+    <a
+      className="social-icon email-btn"
+      href="https://twitter.com/PraisesPJMT"
+      target="_blank"
+      rel="noreferrer"
+    >
+      <FontAwesomeIcon
+        className="social-icon twitter"
+        icon={faTwitter}
+      />
+    </a>
+    <a
+      className="social-icon email-btn"
+      href="https://web.facebook.com/praises.musa"
+      target="_blank"
+      rel="noreferrer"
+    >
+      <FontAwesomeIcon
+        className="social-icon facebook"
+        icon={faSquareFacebook}
+      />
+    </a>
+    <a
+      className="social-icon email-btn"
+      href="https://www.instagram.com/praisespjmt/"
+      target="_blank"
+      rel="noreferrer"
+    >
+      <FontAwesomeIcon
+        className="social-icon instagram"
+        icon={faSquareInstagram}
+      />
+    </a>
```

Comment:
### [Suggestion] 
Please try to add this content with the help of **map-function**. Only you need is to create an array of an object and iterate over each object and render the things dynamically.

Reply:
Sure! I wasn't aware that the JavaScript map method can be used in JSX at the time of writing this project. I just made a research about that and implemented it in a `hotfix` branch.